### PR TITLE
Integration tests upgrade unattached (SC-265)

### PR DESCRIPTION
**Pull request by lucasmoura**
This PR is based on #1738, only dropping the cis check on the integration test were we upgrade from focal into groovy

### CODE REVIEW

**Review by renanrodrigo — APPROVED** [review]
lgtm

File: features/ubuntu_upgrade.feature [hunk 1]
```diff
@@ -13,8 +14,8 @@ Feature: Upgrade between releases when uaclient is attached
         AllowThirdParty=yes
         """
         And I run `sed -i 's/Prompt=lts/Prompt=normal/' /etc/update-manager/release-upgrades` with sudo
-        Then I verify that running `do-release-upgrade <devel_release> --frontend DistUpgradeViewNonInteractive` `with sudo` exits `0`
-        When I reboot the `<release>` machine
+        And I run `do-release-upgrade <devel_release> --frontend DistUpgradeViewNonInteractive` `with sudo` and stdin `y\n`
```
Comment (orndorffgrant) [inline features/ubuntu_upgrade.feature:17]: It looks like this is supposed to use the `stdin` var from the examples?

```suggestion
        And I run `do-release-upgrade <devel_release> --frontend DistUpgradeViewNonInteractive` `with sudo` and stdin `<stdin>`
```

Comment (orndorffgrant) [reply features/ubuntu_upgrade.feature:17]: Or should we leave `y/n` hardcoded here and remove the `stdin` column in the examples?

**Review by orndorffgrant — CHANGES_REQUESTED** [review]
Looks like we can't just delete the cis line. Instead we'll have to specify it is not available.

```
    When I run `ua status` with sudo                                                                                    # features/steps/steps.py:177 0.870s
    Then stdout matches regexp                                                                                          # features/steps/steps.py:361 0.001s
      """
      SERVICE       AVAILABLE  DESCRIPTION
      esm-infra     no       +UA Infra: Extended Security Maintenance \(ESM\)
      """
      Assertion Failed: 
      Expected: a string matching 'SERVICE       AVAILABLE  DESCRIPTION
      esm-infra     no       +UA Infra: Extended Security Maintenance \(ESM\)'
           but: was 'SERVICE       AVAILABLE  DESCRIPTION\ncis           no         Center for Internet Security Audit Tools\nesm-infra     no         UA Infra: Extended Security Maintenance (ESM)\nfips          no         NIST-certified core packages\nfips-updates  no         NIST-certified core packages with priority security updates\nlivepatch     no         Canonical Livepatch service\n\nThis machine is not attached to a UA subscription.\nSee https://ubuntu.com/advantage'
```

**Comment by lucasmoura** [comment]
@orndorffgrant what is the package version you are seeing on `groovy` ?

**Comment by orndorffgrant** [comment]
@lucasmoura I believe that error was from running `tox -e behave-upgrade-21.04` so no groovy involved.

Maybe we should only check for the esm-infra line, instead of both the header and esm-infra line?

Like this
```diff
        Then stdout matches regexp:
        """
-       SERVICE       AVAILABLE  DESCRIPTION
        esm-infra     no       +UA Infra: Extended Security Maintenance \(ESM\)
        """
```

**Comment by lucasmoura** [comment]
@orndorffgrant PR updated

**Review by orndorffgrant — APPROVED** [review]
Sorry for the delay, I ran all the upgrade tests to make sure they were all working. Looks great!

**Comment by orndorffgrant** [comment]
The integration test failure on jenkins is unrelated to these changes.

**Pull request merged**